cmake/Dependencies.cmake

```diff
@@ -1256,6 +1256,9 @@ if (NOT INTERN_BUILD_MOBILE)
   ENDIF()
 
   OPTION(NDEBUG "disable asserts (WARNING: this may result in silent UB e.g. with out-of-bound indices)")
+  IF (${CMAKE_BUILD_TYPE} STREQUAL "Release")
```

Comment:
This won't work for multiple-build-type generators. For better solutions, please refer to https://gitlab.kitware.com/cmake/community/-/wikis/FAQ#make-override-files and https://gitlab.kitware.com/cmake/community/-/wikis/FAQ#dynamic-replace.